.github/workflows/playwright.yml

```diff
@@ -82,7 +80,7 @@ jobs:
       - name: Get Playwright version
         id: playwright-version
         working-directory: apps/site
-        run: echo "version=$(pnpm exec playwright --version | awk '{print $2}')" >> $GITHUB_OUTPUT
+        run: echo "version=$(node_modules/.bin/playwright --version | awk '{print $2}')" >> $GITHUB_OUTPUT
```

Comment:
This might sound weird as we're used to call commands such as npx and whatnot that then pull dependencies outside of local cache. This forcefully uses the local version as if it is not there, something is wrong.